routes/drones.js

```diff
@@ -1,32 +1,49 @@
 const express = require('express');
+// const Drone = require('../models/Drone.model');
 const router = express.Router();
 
+const droneController = require("./../controllers/droneController")
 // require the Drone model here
 
-router.get('/drones', (req, res, next) => {
-  // Iteration #2: List the drones
-  // ... your code here
-});
+router.get("/", droneController.getDrones);
 
-router.get('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
-});
+router.get("/create", droneController.createDrones);
 
-router.post('/drones/create', (req, res, next) => {
-  // Iteration #3: Add a new drone
-  // ... your code here
-});
+router.post("/create", droneController.createDronesForm);
 
-router.get('/drones/:id/edit', (req, res, next) => {
-  // Iteration #4: Update the drone
-  // ... your code here
-});
+router.get("/:droneID/update-form", droneController.editDrone);
 
-router.post('/drones/:id/edit', (req, res, next) => {
-  // Iteration #4: Update the drone
-  // ... your code here
-});
+router.get("/:droneID/update-form", droneController.editDroneForm);
```

Comment:
Recuerda que esta es la ruta a la que le vamos hacer llegar la info pr tanto tiene que ser con el verbo post 😉  y en tu archivo update-form.hbs indicaste que la ruta era:  <form action="/drones/{{drone._id}}/edit" method="post"> entonces tiene que ser de la siguente manera

```suggestion
router.post("/:droneID/edit", droneController.editDroneForm);
```